doc/using_tespy/components.rst

```diff
@@ -54,55 +54,57 @@ Parameters
 
 .. code-block:: python
 
-	from tespy import cmp, hlp
+	from tespy.components import heat_exchanger, pipe
+	from tespy.tools.data_containers import dc_cp
 	import numpy as np
 
-	he = cmp.heat_exchanger('evaporator')
+	he = heat_exchanger('evaporator')
 
 	# ways to specify (and set) value
 	he.set_attr(kA=1e5)
 	# specify data container (same result as above)
-	he.set_attr(kA=hlp.dc_cp(val=1e5, is_set=True))
+	he.set_attr(kA=dc_cp(val=1e5, is_set=True))
 
 	# ways to unset value
 	he.set_attr(kA=np.nan)
 	he.kA.set_attr(is_set=False)
 
 	# custom variables
-	pipe = cmp.pipe('my pipe')
+	pipe = pipe('my pipe')
 
 	# make diameter variable of system
 	pipe.set_attr(D='var')
 	# data container specification with identical result,
 	# benefit: val is the starting value in this case
-	pipe.set_attr(D=hlp.dc_cp(val=0.2, is_set=True, is_var=True))
+	pipe.set_attr(D=dc_cp(val=0.2, is_set=True, is_var=True))
 
 	# data container specification with identical result,
 	# benefit: specification of bounds will increase stability
-	pipe.set_attr(D=hlp.dc_cp(val=0.2, is_set=True, is_var=True, min_val=0.1, max_val=0.3))
+	pipe.set_attr(D=dc_cp(val=0.2, is_set=True, is_var=True, min_val=0.1, max_val=0.3))
 
```

Comment:
This part needs modification: Naming the variable for the component "pipe" will overwrite the import.